doc/lightning-getroute.7.md

```diff
@@ -40,8 +40,10 @@ route generated. 0.0 means the exact fee of that channel is used, while
 default is 5.0, or up to 5% fee distortion.
 
 *exclude* is a JSON array of short-channel-id/direction (e.g. \[
-"564334x877x1/0", "564195x1292x0/1" \]) which should be excluded from
-consideration for routing. The default is not to exclude any channels.
+"564334x877x1/0", "564195x1292x0/1" \]) or node-id which should be excluded
+from consideration for routing. If a node is specified, all the incoming
```

Comment:
No need to specify exact mechanism by which we exclude nodes: as long as the node gets excluded, it is sufficient to just say "the node should be excluded from consideration for routing."

Reply:
Well, it matters in the corner case where we're trying to route from the excluded node I guess.

Reply:
Possibly, but in theory ideally that "should" fail.  Hmm...

Reply:
Now I delete the sentence of "If a node is specified, all the incoming ...".
I also add "Note if the source or destination is excluded, the command result is undefined". :-)